site/2015/05/05/doctrine-orm-module-release-0-9-0.rst

```diff
@@ -0,0 +1,57 @@
+DoctrineORMModule release 0.9.0
+===============================
+
+The **Zend Framework Integration Team** is happy to announce the new release of **DoctrineORMModule**.  
+DoctrineORMModule 0.9.0 is out of door!!  
+
+*Note* This is the last version that supports doctrine\migrations we are working to split this feature in another indipendent module.  
```

Comment:
Typo: `independent`
Replace backslash: `doctrine/migrations`

Also I would reword to `we are working on extracting this feature into another`